simcon_project/users/views/assignment_management.py

```diff
@@ -80,7 +80,7 @@ def get(self, request, *args, **kwargs):
         will be removed as context to the template
         """
         super().get(request, *args, **kwargs)
-        assignment = Assignment.objects.get(pk=self.kwargs['pk'])
+        assignment = Assignment.objects.filter(pk=self.kwargs['pk']).first()
```

Comment:
feels like bad practice to say 
`get me the assignment with this primary key which should be unique, oh the first one that u find` 
no need for `.first()`

Reply:
We use `first()` because it will return an object instead of a queryset